cosmos/launchpad/util.go

```diff
@@ -110,3 +114,17 @@ func toOperations(msg []cosmosclient.Msg, hasError bool) (operations []*types.Op
 	}
 	return
 }
+
+func getFromAddressFromOperations(ops []*types.Operation) (sdk.Address, error) {
```

Comment:
Probably I can clean this later, I added a function that gets this plus other data.